static/css/global.css

```diff
@@ -211,12 +211,17 @@ a:focus {
   display: flex;
   justify-content: space-between;
   padding: 2em;
+  align-items: center;
 }
 
 .navigation__backButton__container {
   background: #fff;
   border: none;
   cursor: pointer;
+  height: 24px;
```

Comment:
Do we need height here? I don't think this changes the styling at all.

Reply:
The height is given to match the size to 24, otherwise the height of the container is 26, which changes the position of the icon inside.